lib/assessment.js

```diff
@@ -5,7 +5,18 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  // get the arguments, assign to args
+  var args = arguments;
 
+  var result = 0;
+
+  // iterate over args length and compare value to result
+  for (var i = 0; i < args.length; i++) {
+    if (args[i] < result) {
+      result = args[i];
+    }
+  }
+  return result;
```

Comment:
This function will return zero if it is passed arguments that are all greater than zero. If you're going to initialize `result` to a value that is not one of the arguments, then you should start it at the largest possible value in JS in order for it to never be less than the arguments. In fact, this is how the built-in `Math.min` works: the initial result value is `Infinity`. Try calling `Math.min` without any arguments!

8/10